tutorials/02_installation.md

```diff
@@ -37,19 +37,16 @@ wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
 On Ubuntu systems, `apt-get` can be used to install `ignition-plugin`:
 ```
 sudo apt-get update
-sudo apt-get install libignition-physics<#>-dev
+sudo apt-get install libignition-physics2-dev
 ```
 
-Be sure to replace `<#>` with a number value, such as `1` or `2`, depending on which version you need.
```

Comment:
I think it's ok to leave as it was, which is easier to maintain in the long run. I believe it was changed on purpose.